R/combineSNV_CNA.R

```diff
@@ -0,0 +1,68 @@
+# combineSNV_CNA.R
+
+#' Combine mutation data from CNA and SNV
+#'
+#' \code{combineSNV_CNA} loads the vector of preprossed dataset from filtered CNA or SNV files,
+#'  creates hash keys corresponding to the input file type, combine and save the hashed
+#'  keys to a gX format file as output file.
+#'
+#' Details.
+#' @section gX files:
+#'  gX files contain data of SNV and CNA combined for each gene as one object, sorted per
+#'  gene. Tab-delimited text file with header.
```

Comment:
I've just changed the spec to make this a data frame RDS. Makes everyone's life easier.